src/utils/consts.ts

```diff
@@ -481,6 +569,28 @@ export const TBTC_ASSET_ADDRESS = THRESHOLD_TBTC_CONTRACTS[CHAIN_ID_ETH].slice(2
 export const THRESHOLD_ARBITER_FEE = 0;
 export const THRESHOLD_NONCE = 0;
 
+export const WORMCHAIN_CONTRACTS_MAINNET = {
+  core: "wormhole1ufs3tlq4umljk0qfe8k5ya0x6hpavn897u2cnf9k0en9jr7qarqqaqfk2j",
+  token_bridge:
+    "wormhole1466nf3zuxpya8q9emxukd7vftaf6h4psr0a07srl5zw74zh84yjq4lyjmh",
+  ibcShimContract:
```

Comment:
lets keep the same naming here, change ibcShimContract to ibc_shim_contract

Reply:
Done!